src/main/resources/com/sonyericsson/rebuild/RebuildAction/BooleanParameterValue.jelly

```diff
@@ -24,6 +24,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
+<?jelly escape-by-default='true'?>
```

Comment:
Plugin-pom run tests to verify that escape-by-default='true' is set for all jelly files.